src/test/unit/spec1.js

```diff
@@ -0,0 +1,26 @@
+describe("app", function () {
+
+  var $rootScope;
+  var $controller;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Variable '$controller' should be initialized on declaration.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664929/issues/source?bid=3233857&fileBranchId=3284028#l4)